src/main/java/com/foundation/salesforce/common/AccessToken.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * @(#) accessToken.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.common;
+
+import io.restassured.response.Response;
+
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
```

Comment:
Please move this import to the second group.

Reply:
Ok I moved that.